omod/src/main/resources/config.xml

```diff
@@ -40,12 +40,27 @@
 	</advice>
 	<!-- /AOP -->
 
+    <globalProperty>
+        <property>patientmatching.configDirectory</property>
+        <defaultValue>patientmatching</defaultValue>
+        <description>The location of the patient matching configuration folder, can be absolute or relative path
+        </description>
+    </globalProperty>
+
 	<globalProperty>
-		<property>patientmatching.linkConfigFile</property>
```

Comment:
Does this mean that existing installations which were using `patientmatching.linkConfigFile` need to manually change it to `patientmatching.configFile`?

Reply:
When we probed around, we don't know of anyone using this module, link is a not an intuitive prefix

Reply:
Renamed it back